src/deploy/RPM_build/RPM.Dockerfile

```diff
@@ -36,7 +35,6 @@ COPY ./binding.gyp .
 COPY ./src/deploy/standalone/noobaa_rsyslog.conf ./src/deploy/standalone/noobaa_rsyslog.conf
 COPY ./src/deploy/standalone/noobaa_syslog.conf ./src/deploy/standalone/noobaa_syslog.conf
 COPY ./src/deploy/standalone/logrotate_noobaa.conf ./src/deploy/standalone/logrotate_noobaa.conf
-COPY ./src/deploy/standalone/logrotate_noobaa.sh ./src/deploy/standalone/logrotate_noobaa.sh
```

Comment:
what about NooBaa.Dockerfile -
https://github.com/noobaa/noobaa-core/blob/1e26ec2ec13cb9a98455e613673283f8207bef52/src/deploy/NVA_build/NooBaa.Dockerfile#L113C51-L114C1

Reply:
I also see it in this test -
https://github.com/noobaa/noobaa-core/blob/1e26ec2ec13cb9a98455e613673283f8207bef52/src/test/unit_tests/test_debug_module.js#L93

Reply:
@guymguym these two files are not part of NC NSFS logrotate, This code was there before

Reply:
ok, but is there any reason why it should be different there? It just adds confusion leaving some cases still use the extra noobaa dir...